### DISCUSSION TITLE
How resize and zoom pre- and post- transform would work to keep same dimension?

### DISCUSSION THREAD

**Discussion by OmarAshkar** [Project-MONAI/MONAI#6134] [Q&A]
Hello,

I tried to run inference on WSI, but the process is extermely slow. Now I am thing to downsample my data. I am wondering if Resize then Zoom transform would keep the same pixel locations to the original place or not?

Thanks in advance!

**Comment by KumoLiu** [comment]
Hi @omashkartrx, what's your pixel locations mean? Both of these transforms change the resolution of the output image if you are referring to the resolution.
Additionally, they are both using `torch.nn.interpolate`, for more details you could also see https://pytorch.org/docs/stable/generated/torch.nn.functional.interpolate.html.

Hope it can help you, thanks!
